js/deribit.js

```diff
@@ -1457,6 +1457,16 @@ module.exports = class deribit extends Exchange {
         return this.safeString (timeInForces, timeInForce, timeInForce);
     }
 
+    parseOrderType (orderType) {
```

Comment:
@DoctorSlimm, I suspect that you unintentionally added changes from deribit here, can you revert it?